model-optimizer/extensions/front/tf/floor_div.py

```diff
@@ -0,0 +1,57 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+from extensions.ops.activation_ops import Floor
+from extensions.ops.elementwise import Mul, Pow, Div
+from mo.front.common.replacement import FrontReplacementPattern
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph, Node, rename_node
+
+
+class FloorDiv(FrontReplacementPattern):
```

Comment:
Please, rename the transformation. Someday when we add support for the operation FloorDiv this transformation name will be confusing. Let's call it like `FloorDivDecomposition`. The same comment for the file name.